.github/workflows/pr-test.yml

```diff
@@ -22,7 +22,9 @@ name: pr-test
 
 on:
   push:
-    branches-ignore: [ master, main ]
+    branches-ignore:
+      - master
+      - main
```

Comment:
Wait, why does this even run in other PRs? Because those PRs are opened by dependabot and the topic branches are pushed directly into gitgitgadget/gitgitgadget!

So it is my mistake: I should not have made those checks mandatory for PRs...

Reply:
> So it is my mistake: I should not have made those checks mandatory for PRs...

I do not understand what you are saying.  This workflow is for push requests.  How could it be done for PRs?

Reply:
Indeed, it is for pushes only! So it _won't work_ to make this a required check to pass for PRs. Yet that's exactly what I had done.

It _happened_ to work for the dependabot PRs because dependabot pushes directly into gitgitgadget/gitgitgadget. Therefore, there was a push, and there was a check that passed.

But it did not work for your PR because you pushed to webstech/gitgitgadget, and the check from over there is not seen by the PR checks in gitgitgadget/gitgitgadget.

I fixed that by not requiring those checks anymore.

Reply:
> I fixed that by not requiring those checks anymore.

This is the part I was missing.  A search showed me where to find the required checks settings.  I was looking for rules in all the wrong places (or something like that).